Assets/AnkrSDK/Examples/UseCases/UseCaseUI.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace AnkrSDK.UseCases
+{
+	public class UseCaseUI : MonoBehaviour
+	{
+		public Button _selectButton;
```

Comment:
Unity serialized fields should not be public. If you need to access it from outside use get only property and [SerializeField] private